packages/falcon-magento2-api/src/authorization.ts

```diff
@@ -1,10 +1,11 @@
-const deepMerge = require('deepmerge');
-const OAuth = require('oauth');
+import deepMerge from 'deepmerge';
+import OAuth from 'oauth';
```

Comment:
```suggestion
import { OAuth } from 'oauth';
```

and on this line:

https://github.com/deity-io/falcon/blob/c814cf32f083b593a73b0cb770acaf3435065710/packages/falcon-magento2-api/src/authorization.ts#L75

replace with `new OAuth(...)`

Reply:
done